client/src/Components/CategoriesList.jsx

```diff
@@ -0,0 +1,126 @@
+import { useState, useEffect } from "react";
+import { AiOutlineCaretDown } from "react-icons/ai";
+import { AiOutlineCaretUp } from "react-icons/ai";
+function CategoriesList() {
+  const [categories, setCategories] = useState([]);
+  useEffect(() => {
+    fetch("https://api.coingecko.com/api/v3/coins/categories")
+      .then((response) => response.json())
+      .then((data) => setCategories(data))
+      .catch((error) => console.error(error));
+  }, []);
+  console.log(categories);
+
+  return (
+    <div className="overflow-x-auto">
+      <table className="table" style={{ width: "1400px" }}>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Market Cap</th>
+            <th>Top Gainer</th>
+            <th>Volume</th>
+            <th>23hr volume</th>
+            <th>Top 3</th>
+          </tr>
+        </thead>
+        <tbody>
+          {categories.map((crypto) => {
+            return (
+              <tr>

```

Comment:
could you change this to <tr key={crypto.id}> ?
it seems like react requires you to have a key attribute